### PR TITLE
[Draft] Adds handling of "take_screenshot" commands

### PR DESCRIPTION
**Pull request by ferrolho**
This goes in hand with [rdeits/MeshCat.jl#195](https://github.com/rdeits/MeshCat.jl/pull/195).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -985,6 +986,9 @@ class Viewer {
             this.set_animation(cmd.animations, cmd.options);
         } else if (cmd.type == "set_control") {
             this.set_control(cmd.name, cmd.callback, cmd.value, cmd.min, cmd.max, cmd.step);
+        } else if (cmd.type == "take_screenshot") {
```
Comment (rdeits) [inline src/index.js:989]: Could you rename this to `save_image` as well, to reflect the fact that the image is saved to disk? Someday I'd also like to enable transmitting the image back to the Julia client, so I'd like to make sure that the behavior implemented here has a clearly identifiable name.

**Review by rdeits — COMMENTED** [review]
Thanks! Just one minor request.

**Comment by ferrolho** [comment]
Done :slightly_smiling_face:

**Comment by rdeits** [comment]
Thanks!

>Actually, should I have committed dist/main.min.js? 

Yes--it's annoying to have to commit it manually, but it makes this package easier to use downstream.

**Pull request merged**